### PR TITLE
Misc superficial doc and formatting minutiae

### PR DESCRIPTION
**Pull request by tkelman**
with apologies for the OCD

### CODE REVIEW

File: doc/manual/types.rst [hunk 1]
```diff
@@ -120,8 +120,8 @@ using :func:`convert`:
 .. doctest:: foo-func
 
     julia> function foo()
-             x::Int8 = 100
-             x
+               x::Int8 = 100
+               x
            end
```
Comment (omus) [inline doc/manual/types.rst]: Should all examples in the manual be indented with 4 spaces? I've noticed indentation can be rather inconsistent.

Comment (tkelman) [inline doc/manual/types.rst]: That's the style guideline for base code, so I think picking something for consistency in doc code examples makes sense?

Comment (omus) [inline doc/manual/types.rst]: Makes sense to me

**Comment by StefanKarpinski** [comment]
Thanks for this, @tkelman. Doing it in a batch like this definitely streamlines the process.

**Pull request merged**